entrypoint.sh

```diff
@@ -11,15 +11,25 @@ fi
 config=`echo $configs | head -n 1`
 interface="${config%.*}"
 
+if [[ "$(cat /proc/sys/net/ipv4/conf/all/src_valid_mark)" != "1" ]]; then
+    echo "sysctl net.ipv4.conf.all.src_valid_mark=1 is not set" >&2
+    exit 1
+fi
+
+sed -i "s:sysctl -q net.ipv4.conf.all.src_valid_mark=1:echo skipping setting net.ipv4.conf.all.src_valid_mark:" /usr/bin/wg-quick
```

Comment:
Just to double-check, would you mind responding to this comment with the before and after of `wg-quick`?

Reply:
Before: https://termbin.com/926v
After: https://termbin.com/rcod

Diff:
```
@@ -237,7 +237,7 @@
        printf -v restore '%sCOMMIT\n*mangle\n-I POSTROUTING -m mark --mark %d -p udp -j CONNMARK --save-mark %s\n-I PREROUTING -p udp -j CONNMARK --restore-mark %s\nCOMMIT\n' "$restore" $table "$marker" "$marker"
        printf -v nftcmd '%sadd rule %s %s postmangle meta l4proto udp mark %d ct mark set mark \n' "$nftcmd" "$pf" "$nftable" $table
        printf -v nftcmd '%sadd rule %s %s premangle meta l4proto udp meta mark set ct mark \n' "$nftcmd" "$pf" "$nftable"
-       [[ $proto == -4 ]] && cmd sysctl -q net.ipv4.conf.all.src_valid_mark=1
+       [[ $proto == -4 ]] && cmd echo skipping setting net.ipv4.conf.all.src_valid_mark
        if type -p nft >/dev/null; then
                cmd nft -f <(echo -n "$nftcmd")
        else
```

Reply:
Perfect! Thanks!